python/hail/docs/scripts/setupTutorial.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+###################################################################
+# Downloads tutorial files if no directory is given
+# Makes symlinks if necessary
+###################################################################
+
+set -e
+
+tutorialFiles=$1
+
+if [ -d 1kg.vds/ ]; then
+    rm -r 1kg.vds
+fi
+
+if [ -z $tutorialFiles ]; then
```

Comment:
You should probably use double quotes here:

```bash
if [ -z "$tutorialFiles" ]
```

It's not always clear to me why and when they are necessary, but they never hurt.